crates/volta-core/src/layout/windows.rs

```diff
@@ -0,0 +1,47 @@
+use std::path::PathBuf;
+
+use super::{volta_home, volta_install};
+use crate::error::ErrorDetails;
+use cfg_if::cfg_if;
+use volta_fail::Fallible;
+
+pub(super) fn default_home_dir() -> Fallible<PathBuf> {
+    let mut home = dirs::data_local_dir().ok_or(ErrorDetails::NoLocalDataDir)?;
+    home.push("Volta");
+    Ok(home)
+}
+
+cfg_if! {
+    if #[cfg(test)] {
+        pub(super) fn default_install_dir() -> Fallible<PathBuf> {
+            Ok(PathBuf::from(r#"Z:\"#))
```

Comment:
Worth a comment explaining how the tests use `Z:\`?

Reply:
The tests currently don't actually need a path that exists, but they need the function to not crash because there aren't valid values in the registry. I'll add a comment to that effect :+1: